libs/financial-aid/shared/src/lib/const.ts

```diff
@@ -1,6 +1,18 @@
+export const formRoutes = '/umsokn/'
+
 export const Routes = {
   application: '/umsokn',
-  spouseApplication: '/umsokn/rettur',
+  form: {
+    info: `${formRoutes}rettur`,
+    relationShip: `${formRoutes}hjuskaparstada`,
+    homeCircumstances: `${formRoutes}buseta`,
+    hasIncome: `${formRoutes}tekjur`,
+    employment: `${formRoutes}atvinna`,
+    usePersonalTaxCredit: `${formRoutes}personuafslattur`,
+    bankInfo: `${formRoutes}bankaupplysingar`,
+    contactInfo: `${formRoutes}samskipti`,
+    conformation: `${formRoutes}stadfesting`,
+  },
```

Comment:
This is nice, we should maybe think about doing something similiar for other parts of our application